.github/workflows/full_stack_tests.yml

```diff
@@ -104,7 +104,8 @@ jobs:
           VIDEO_RECORDING_IS_ENABLED=0
           xvfb-run -a --server-args="-screen 0, 1285x1000x24"
           python -m scripts.run_e2e_tests --skip-install
-          --skip-build --suite=${{ matrix.suite.name }} --prod_env
+          --skip-build --suite=${{ matrix.suite.name }} --prod_env &&
```

Comment:
Why `&& exit 1`? A comment might be helpful.

Reply:
It was for testing. I have removed it.